### DISCUSSION TITLE
Disable magic link

### DISCUSSION THREAD

**Discussion by nullsumme** [outline/outline#3333] [Self Hosting] [answered]
Hey,

I've got a setup running with OIDC auth. Ist there any way to disable the magic link auth? Its enabled an I can't disable it.

![Bildschirmfoto 2022-04-04 um 16 18 34](https://user-images.githubusercontent.com/369577/161563926-646d2b5f-2263-4805-985e-7595d25b5150.png)
![Bildschirmfoto 2022-04-04 um 16 19 37](https://user-images.githubusercontent.com/369577/161564107-79052e00-9d92-4a4a-a9ec-c350b22fdae6.png)
.

**Comment by nullsumme** [comment, marked as answer]
Okay. I pretty sure that's a bug.
If I set "SMTP_HOST=something" I can disable it. If SMTP_HOST is not set, it is enabled by default and I can't disable it. 

Maybe the double negation is wrong?
env.ts:
`EMAIL_ENABLED: !!env.SMTP_HOST || env.NODE_ENV === "development",`

**Reply by tommoor** [reply]
Yea, it's a bug but it has been fixed for the next version 👍
